ql/src/test/org/apache/hadoop/hive/ql/optimizer/calcite/rules/TestRuleHelper.java

```diff
@@ -88,16 +83,16 @@ public static RelBuilder buildRelBuilder(AbstractRelOptPlanner planner,
     return HiveRelFactories.HIVE_BUILDER.create(optCluster, schemaMock);
   }
 
-  static RexNode eq(RelBuilder relBuilder, String field, Number value) {
+  public static RexNode eq(RelBuilder relBuilder, String field, Number value) {
     return relBuilder.call(SqlStdOperatorTable.EQUALS,
         relBuilder.field(field), relBuilder.literal(value));
   }
 
-  static RexNode or(RelBuilder relBuilder, RexNode... args) {
+  public static RexNode or(RelBuilder relBuilder, RexNode... args) {
     return relBuilder.call(SqlStdOperatorTable.OR, args);
   }
 
-  static RexNode and(RelBuilder relBuilder, RexNode... args) {
+  public static RexNode and(RelBuilder relBuilder, RexNode... args) {
```

Comment:
The `RelBuilder` API has methods for doing this operations so we don't need to use these utilities.

Reply:
Thanks, I will use the APIs from RelBuilder instead.